etc/a.gwf

```diff
@@ -280,12 +318,24 @@ public_if_titles=no
 # "yes" = consider; "no" = don't consider
 public_if_no_date=no
 
+# Governs the behaviour of the person_exist function
+# off -> true
+# _ ->
+#       match person_of_key base fn sn oc with
+#         Some ip -> authorized_age conf base (pget conf base ip)
+#       | None -> false
+#red_if_not_exist=off/on
```

Comment:
I will cretainly remove this feature as well.

Reply:
Dont  remember why it exists! Ok to remove